Sites/GD2/Project 1/Styles.css

```diff
@@ -0,0 +1,50 @@
+body {
+    margin: 100px;
+}
+
+h1 {
+    font-family: 'Noto Sans TC', sans-serif;
+    color: #260648;
+    font-weight: bold;
+    font-size: 45px;
+}
+
+h2 {
+    font-family: 'Noto Sans TC', sans-serif;
+    color: #6B4B8C;
+    font-weight: normal;
+    font-size: 30px;
+    text-align: left;
```

Comment:
By default, most elements, including the `h2` have a left alignment. You can safely remove this line.